free-1.md

```diff
@@ -44,6 +51,7 @@ _Are there any other DSLs in this domain? If not, describe how you know there
 aren't and conjecture why not. If so, describe them and provide links. How well 
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
+I dont think there any other DSLs in this domain, but I definitely could be wrong. I looked a fair amount on google and couldnt find anything but I also couldnt think of a really good search to do. It seems obvious that there should be this, so I dont know why there isnt (if in fact there isnt)
```

Comment:
I think the distinction is that Siri/Google Now/Cortana fit the general purpose language since they aren't only for creating schedules. However, I would argue that Google Calendar is a DSL in this domain.